script.js

```diff
@@ -0,0 +1,109 @@
+const name = document.querySelector('.name');
+const phone = document.querySelector('.phone');
+const select = document.querySelector('.regions');
+const selectCity = document.createElement('select');
+const checkbox = document.querySelector('.check');
+let label = document.createElement('label');
```

Comment:
create the label where you need it. currently the logic is too much spread across the file.